lib/tli.rb

```diff
@@ -130,6 +130,12 @@ def parse_options(args)
       fail "#{arg}: not a valid option" unless OPTIONS.include?(sym)
       if OPTIONS[sym] == :key_value
         fail "#{arg} requires a value" if index + 1 >= length
+
+        if (sym == :source || sym == :target) &&
+           !TRANSLATORS['google'].langs.include?(args[index + 1])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.